packages/block-editor/src/store/reducer.js

```diff
@@ -192,6 +195,154 @@ export function isUpdatingSameBlockAttribute( action, lastAction ) {
 	);
 }
 
+/**
+ * Higher-order reducer intended to reset the cache key of all blocks
+ * whenever the post meta values change.
+ *
+ * @param {Function} reducer Original reducer function.
+ *
+ * @return {Function} Enhanced reducer function.
+ */
+const withPostMetaUpdateCacheReset = ( reducer ) => ( state, action ) => {
+	const newState = reducer( state, action );
+	const previousMetaValues = get( state.settings.__experimentalMetaSource, [ 'value' ] );
```

Comment:
An error I'm seeing locally, which I'm surprised is not occurring in Travis (maybe a difference of preferences persistence cache): `state` will be `undefined` here initially, right? So `state.settings` will in-fact be problematic to try to access. I guess you might need your previous `get` usage 😅 (at least at this line, I think the following one should be okay as-is)